### DISCUSSION TITLE
OB-6: Sending patch silences device

### DISCUSSION THREAD

**Discussion by wharfrat83** [christofmuc/KnobKraft-orm#336] [Q&A]
Hi all - 

I just discovered Knob Kraft and have been loving it with my Prophet 10. Sadly, I'm encountering some serious issues using it with my OB-6 (Desktop module, connected with DIN midi via MioXL midi interface, M1 MacBook Pro). 

It works perfectly with the Prophet - patches send without issue. With the OB-6, any time I try to send a patch from KnobKraft, the entire synth goes silent, despite KnobKraft's midi status monitor showing the patch being sent successfully. 

Once I send the patch from KnobKraft, NO patches on the OB-6 make any sound. Not the patch I was on (e.g. 001) or any other patches I select thereafter. 

The only way to get sound again is by fully powering the device off and back on. 

I have compared all the device/midi settings to the Prophet (midi channel, param xmit, receive, etc.). They are identical. But something is getting seriously corrupted on the OB-6. 

If anyone has any ideas, I would be hugely appreciative. Thanks!

**Comment by christofmuc** [comment]
Sorry to hear you're having problems. The one thing I could think of is that the OB-6 has additionally implemented the settings page, where you can edit the Synth's main settings like MIDI channel. Did you check the settings page that everything is fine?

All patches going quiet usually is MIDI volume that gets suprisingly set by something. The MIDI Log of the KnobKraft shows the exact messages it sends. Can you do the following checks:

1. Check the Settings page of the OB-6 (in Knobkraft) to make sure everything is sane.
2. Set the send mode (selector underneath the button grid) to "Sysex" to make sure you're transferring the patch data and not just a program change.
3. Select a patch that makes the synth go quiet, and save the full MIDI log and attach it here. We need to look at the outgoing data to figure out what is going wrong. 

Thank you!
